### PR TITLE
add the impact of updating and deleting jobflow and jobtemplate. Stat…

### PR DESCRIPTION
**Pull request by zhoumingcheng**
add the impact of updating and deleting jobflow and jobtemplate. Status switching, switching conditions and impact of jobflow

### CODE REVIEW

**Review by jiangkaihua — COMMENTED** [review]
/lgtm

File: docs/design/jobflow/README.md [hunk 1]
```diff
@@ -48,7 +48,13 @@ jobflow job submission complete process：
 
 6. After assigning nodes to Pods, kubelet will get the Pod's configuration from kube-apiserver and start the corresponding containers.
 
+**update jobflow**:
 
+Currently, jobflow does not support the update operation, and the update of jobflow will be blocked through webhook.
+
+**delete jobflow**:
+
+After deleting jobflow, all vcjobs created by jobflow will be deleted directly, no matter what state the vcjobs are in.
```
Comment (hwdef) [inline docs/design/jobflow/README.md:57]: It is recommended to use webhook interception here. If there is a running job, it is not allowed to delete the jobflow. Unless the --force parameter is used when removing

File: docs/design/jobflow/README.md [hunk 2]
```diff
@@ -289,7 +299,19 @@ Here is an example of jobflow:
...
+
+Create a jobtemplate to be used by jobflow.
+
+**update jobtemplate**:
+
+After the jobtemplate is updated, it will not affect the vcjobs that have been created based on the jobtemplate. It will not affect the successfully executed jobflow. It may affect the jobflow that has not been executed. For example, the jobflow that has not been executed to the jobtemplate stage will use the updated jobtemplate template.
+
+**delete jobtemplate**:
+
+After the jobtemplate is deleted, it will not affect the vcjob that was originally created based on the jobtemplate. It will not affect the successfully executed jobflow. It may affect the jobflow that has not been executed. For example, the jobflow that has not been executed to the jobtemplate stage will not find the jobtemplate template and cannot proceed to the next step.
```
Comment (hwdef) [inline docs/design/jobflow/README.md:314]: Something should be done here, such as:
It is not allowed to delete the jobtemplate when there is jobflow using this jobtemplate

Comment (zhoumingcheng) [reply docs/design/jobflow/README.md:314]: I agree with this comment, I will add this judgment in the webhook module

File: docs/design/jobflow/README.md (hunk 1 above)
Comment (zhoumingcheng) [reply docs/design/jobflow/README.md:57]: Thank you, I will add it in the webhook. When you need to delete the jobflow in the running state, you can only delete it with the command kubectl delete vcjob {vcjobName} --force

**Review by hwdef — COMMENTED** [review]
/lgtm

**Comment by Thor-wl** [comment]
@zhoumingcheng Is is PR ready for review?

**Comment by zhoumingcheng** [comment]
> Is is PR ready for review

**Comment by Thor-wl** [comment]
> yes, thanks.

OK. I'll review it later. Really appricate this work.

**Review by william-wang — APPROVED** [review]
/approve

**Pull request merged**